yoti-sdk-spring-boot-example/src/main/java/com/yoti/api/examples/springboot/IdentitySessionController.java

```diff
@@ -37,8 +37,8 @@ public IdentitySessionController(DigitalIdentityClient client) {
     public String identityShareSession() {
         ShareSession session = client.createShareSession(
                 // forMinimalShare()
-                forDynamicScenarioShare()
-                // forIdentityProfileShare()
+                // forDynamicScenarioShare()
+                forIdentityProfileShare()
                 // forLocationExtensionShare()
         );
```

Comment:
I have no idea what that does, but is that normal? Just checking it is not a mistake

Reply:
these are different inputs u can use in the example project to test different shares. It is used as a playground by our QA and maybe integrators